test/hotspot/jtreg/runtime/Thread/StopAtExit.java

```diff
@@ -72,6 +72,8 @@ public static void main(String[] args) {
                 usage();
             }
         }
+        timeMax /= 2;  // Split time between the two sub-tests.
```

Comment:
Maybe bump DEF_TIME_MAX to 60 seconds now so both subtests run the original 30 seconds? My intention was to run the same test again(same time) but with the addition of threadCreator.

Reply:
I try not to have the default config for a stress test run longer that 30 seconds total.
15 seconds per sub-test is fine.